tests/lib/rules/indent.js

```diff
@@ -1428,7 +1441,7 @@ ruleTester.run("indent", rule, {
         {
             code: fixture,
             output: fixedFixture,
-            options: [2, {SwitchCase: 1}],
+            options: [2, {SwitchCase: 1, MemberExpression: 1}],
```

Comment:
This inconsistency seems to exist throughout the file.

For example, `SwitchCase` without a space happens on lines [266](https://github.com/eslint/eslint/blob/288f7bf47279076e6b3f05814ebec3f6b7e5aeb6/tests/lib/rules/indent.js#L266), [537](https://github.com/eslint/eslint/blob/288f7bf47279076e6b3f05814ebec3f6b7e5aeb6/tests/lib/rules/indent.js#L537), [542](https://github.com/eslint/eslint/blob/288f7bf47279076e6b3f05814ebec3f6b7e5aeb6/tests/lib/rules/indent.js#L542), and ten additional places.

 But `SwitchCase` with a space occurs in lines [776](https://github.com/eslint/eslint/blob/288f7bf47279076e6b3f05814ebec3f6b7e5aeb6/tests/lib/rules/indent.js#L776) and [1897](https://github.com/eslint/eslint/blob/288f7bf47279076e6b3f05814ebec3f6b7e5aeb6/tests/lib/rules/indent.js#L1897).

If the inconsistency is something that should be fixed eventually, maybe I can leave as is for this PR and I or someone else can open up a separate issue about this? Might be a good one to apply the `beginner` label to.